src/areaCalculator/AreaCalculator.java

```diff
@@ -1,11 +1,50 @@
 package areaCalculator;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class AreaCalculator {
 
-    public double calculateAreaOfRectangle(Rectangle rectangle) {
-        double area = rectangle.getLength() * rectangle.getWidth();
-        System.out.println("Area of rectangle (" + rectangle.getLength() + " x "
-                + rectangle.getWidth() + ") = " + area);
-        return area;
+    private List<Shape> shapeList;
+
+    public AreaCalculator() {
+        shapeList = new ArrayList<>();
+    }
+
+    void addShapeToList(Shape shape) {
+        shapeList.add(shape);
+    }
+
+    public void calculateArea(List<Shape> list) {
```

Comment:
If you introduced shapeList field in this class, I think you should operate on it, instead of accepting a list of shapes from outside the class.
Of course you can leave it as is, but then the field (shapeList) makes no sense so one of two should be removed.

Reply:
Removed method`s `calculateArea` input parameter